applications/registration-react/src/redux/modules/user.js

```diff
@@ -0,0 +1,53 @@
+import { compose } from 'recompose';
```

Comment:
Additional dependency could be avoided as enhancer functions can be combined without it